Payments/Operation/03_ManageRefunds.md

```diff
@@ -0,0 +1,77 @@
+> [Info] This means that you cannot follow up a case with the ID of an authorization or a payments and captures transaction. However, the most payment service providers support an original reference ID that you can use to search for a specific case in the whole payment process.
+
+You can use this procedure to display all incoming references to a refund from a payment service provider. You can check for communication errors between Actindo and the payment service provider that has been caused in a termination of a workflow, for example. In addition, if applicable, you can manually change the status of a transaction to *Void* to clear the database.    
+   
+If everything works fine with the posting of a refund, you do not need to monitor the *Refunds* menu entry. You can follow up the successfully processed refunds with the *Confirmation* menu entry.
+
+## Check refund transaction
+
+A refund transaction provides numerous payment-related details, which may vary depending on the payment service provider settings. To decide whether you can manually void a refund transaction, you need to check its details.   
+> [Info] You can only change the status of a refund transaction, but not any data.
+
+#### Prerequisites
+
+- At least one connection has been created, see [Create a connection](../Integration/01_ManageConnections.md#create-a-connection).
+- At least one refund has been created.
+- It is recommended that you add the *Status ID* field to your view so that you are able to sort or filter for specific statuses. 
+
+#### Procedure   
+
+*Payments > Refunds > Tab LIST* 
+
+![Refund transactions](../../Assets/Screenshots/Payments/Refunds/LISTRefunds.png "[Refund transactions]")
+
+1. Check the refund transaction status in the *Status* column. The status displays the current stage in the payment process. You can use the status ID prefixed below to filter the list. The following statuses are available:   
+    - **1 - Preparing**  
+        The transaction has been created, but is not yet transferred.
+    - **2 - Unconfirmed**   
+        A refund is reserved but not yet released by an Actindo user. This status is relevant for some functions only, for example, the cancellations and chargebacks.
+    - **3 - Success**  
+        Actindo has got the response from the service payment provider, that the transfer was successfully.   
+        You can now follow up this case under the menu entry *Confirmations*. <!-----Stefan ist das richtig?---> 
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the status information field provides further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void a refund ticket](#void-a-refund-transaction)
```

Comment:
```suggestion
       The transaction has been voided, see [Void refund transaction](#void-refund-transaction)
```
Grundsätzlich zu den Headern und Links: Wir haben in der Regel bei Plural in Header 1 kein Artikel verwendet (z.B. Manage refunds), aber dann bei den weiteren Headern/Sub-procedures  mit Artikel (z.B. Void a refund transaction). Hier ist es gemischt. Prüfe mal bitte alle durchgängig und passe sie alle an (achte auch bitte darauf, dass es bei den kleinsten Abweichungen -z.B. ohne oder mit Artikel- der Link nicht mehr funktioniert).

Reply:
done